# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-rust-adler2.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-144.0.7559.59/build/rust/std/BUILD.gn.vanilla	2026-01-14 18:04:58.911037837 +0100
+++ chromium-144.0.7559.59/build/rust/std/BUILD.gn	2026-01-14 18:05:47.164616569 +0100
@@ -55,11 +55,7 @@
       "miniz_oxide",
       "object",
     ]
-    if (rustc_nightly_capability) {
-      stdlib_files += [ "adler2" ]
-    } else {
-      stdlib_files += [ "adler" ]
-    }
+    stdlib_files += [ "adler2" ]    
   }
 
   if (toolchain_for_rust_host_build_tools) {
